### ISSUE TITLE
Convert NCR test to SystemTestsCompareTwo infrastructure

### ISSUE THREAD

**Issue by billsacks** [ESMCI/cime#444]
We want to convert the various system tests that compare two runs to use the SystemTestsCompareTwo infrastructure, where possible. I am opening separate issues for each such system test, so that we can check them off as we do this conversion.

**Comment by jgfouca** [comment]
Assigning @mfdeakin-sandia since he's done some of these in the past.

**Comment by rljacob** [comment]
Should probably give this to someone else since Michael is committed to critical path team tasks.

**Comment by mfdeakin-sandia** [comment]
I think I have some time coming up; they didn't mention any SE things in the meeting yesterday

**Comment by rljacob** [comment]
ok.

**Comment by billsacks** [comment]
I stumbled across what I think is a bug in the current NCR: I _think_ that

```python
restore("env_build.NCR2.xml", "env_build.xml")
```

should be

```python
restore("env_build.NCR2.xml", newname="env_build.xml")
```

**Comment by mvertens** [comment]
@billsacks - I added this fix in PR #1638.
